.clippy.toml

```diff
@@ -1 +1,6 @@
 upper-case-acronyms-aggressive = true
+
+# only intended to affect main rustls crate - need to allow disallowed-types in all other crates in Clippy CI job
+disallowed-types = [
+  { path = "std::sync::Arc", reason = "must use Arc from sync module to support downstream forks targetting architectures without atomic ptrs" },
+]
```

Comment:
I was hoping to configure this `disallowed-types` *only* for the main rustls crate & remove the workaround I made to ignore this for all other crates in CI. I tried configuring this in `[lints.clippy]` in `rustls/Cargo.toml` but to no avail. Any ideas or pointers?